downstream/modules/hub/proc-delete-collection.adoc

```diff
@@ -7,16 +7,19 @@ You can further manage your collections by deleting unwanted  collections, provi
 
 .Prerequisites
 * The collection being deleted does not have dependencies with other collections.
-* You have *Delete Collections* permissions
+* You have *Delete Collections* permissions.
 
 .Procedure
 . Log in to {PlatformName}.
 . Navigate to menu:Automation Hub[Collections].
 . Click a collection to delete.
-. Click the btn:[More Actions] icon *{MoreActionsIcon}* then select an option:
+. Click the btn:[More Actions] icon *{MoreActionsIcon}*, and then select an option:
 .. *Delete entire collection* to delete all versions in this collection.
-.. *Delete version [number]* to delete the current version of this collection. You can change versions using the *Version* dropdown menu.
+.. *Delete version [number]* to delete the current version of this collection. You can change versions using the *Version* drop-down menu.
```

Comment:
Perhaps consider "by using" rather than "using" for more clarity.

IBM style snippet: "To avoid ambiguity, replace this gerund with either by using or that uses."